manifests/integrations/nginx.pp

```diff
@@ -24,6 +24,7 @@
 #
 class datadog_agent::integrations::nginx(
   Array $instances = [],
+  $logs = undef,
```

Comment:
Should be an `Optional` `Array`, you can even go as far as specifying an `Array` of `Hashes`, but `Array` is probably enough :)

The default to `undef` is 👌